documents/data-sharing.md

```diff
@@ -0,0 +1,58 @@
+# Annual survey data sharing
```

Comment:
Can we refer to it as the "annual community survey" (also in the name of the file). We are likely to do many more types of surveys and I believe we want to limit the scope to just the annual community survey. We may also additionally want to link to the 2021 survey as an example.